sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -196,7 +196,9 @@ public void serialize(StoreResult storeResult,
             jsonGenerator.writeNumberField("itemLSN", storeResult.itemLSN);
             jsonGenerator.writeStringField("sessionToken", (storeResult.sessionToken != null ? storeResult.sessionToken.convertToString() : null));
             jsonGenerator.writeStringField("exception", BridgeInternal.getInnerErrorMessage(storeResult.exception));
-            jsonGenerator.writeObjectField("transportRequestTimeline", storeResult.storeResponse != null ? storeResult.storeResponse.getRequestTimeline() : null);
+            jsonGenerator.writeObjectField("transportRequestTimeline", storeResult.storeResponse != null ?
```

Comment:
Can we add a small test case by getting error and checking for request timeline in diagnostics. 404 or wrong Pk will do the work ?

Reply:
good suggestion. I will include the test in the PR. thanks @simplynaveen20

Reply:
added.